s3tests/functional/test_s3.py

```diff
@@ -5351,9 +5351,15 @@ def test_multipart_copy_invalid_range():
     bucket, key = _create_key_with_random_content('source', size=5)
     upload = bucket.initiate_multipart_upload('dest')
     e = assert_raises(boto.exception.S3ResponseError, copy_part, bucket.name, key.name, bucket, 'dest', upload.id, 0, 0, 21)
-    eq(e.status, 400)
-    eq(e.reason, 'Bad Request')
-    eq(e.error_code, 'InvalidArgument')
+    # on RHEL we get response 416, check bugzilla 1501005
```

Comment:
not necessary to mention rhel or bugzillas, all versions of radosgw behave this way

Reply:
@cbodley if all versions behave same, do we need to check for 400 status code? can it be only 416 now? Not sure why it is 400?

Reply:
that's how it was submitted in https://github.com/ceph/s3-tests/pull/179, presumably because the [referenced s3proxy change](https://github.com/gaul/s3proxy/pull/238) returned 400 there